### PR TITLE
Gus/multiple upload

### PR DESCRIPTION
**Pull request by motizuki**
Acceptance Criteria
=================
Allow users to drag n drop multiple images from dashboard.
For each image, a new draft instance of a resource needs to be created
Dashboard design and implementation based on Lucie's prototypes

Background
=================
Using Dropzone.js to manage dragdrop feature.

Checklist
=================

- [ ] Code is understandable without Dev
- [ ] Acceptance criteria, what was changed, and why it was changed (If applicable) on Pull / Merge Request
- [ ] All Tests are Passing
- [ ] Code Coverage goes up
- [ ] All new methods have a relevant spec
- [ ] Methods have a single responsibility (Where applicable)
- [ ] Linters Pass
- [ ] ‘Yard’ style comments on methods and classes (Where applicable)

### CODE REVIEW

File: app/controllers/dam_dashboard_controller.rb [hunk 1]
```diff
@@ -3,16 +3,17 @@ class DamDashboardController < ApplicationController
...
-      @dam_resources = DamResource.under_review.paginate(page: params[:page], per_page: 10) 
-      @table_title = "Items to be approved"
-    end
-    if current_dam_user.user?
-      @dam_resources = DamResource.find_by_status([:draft, :rejected])
-                                  .where(dam_user_id: current_dam_user.id)
-                                  .paginate(page: params[:page], per_page: 10) if current_dam_user.user?
-      @table_title = "Items requiring action"
-    end
+    @under_review_dam_resources = DamResource.under_review
```
Comment (hapiben) [inline app/controllers/dam_dashboard_controller.rb]: Is it possible to have this as a scope? Something like [this](http://craftingruby.com/posts/2015/06/24/say-no-to_chained-scopes.html).

File: app/assets/stylesheets/dam_uploader/dam_resource_show.scss [hunk 2]
```diff
@@ -13,8 +14,3 @@
     }
```
Comment (hapiben) [inline app/assets/stylesheets/dam_uploader/dam_resource_show.scss]: Orphaned bracket?

File: app/helpers/dam_uploader_helper.rb [hunk 3]
```diff
@@ -24,4 +24,20 @@ def authentication_button
         class: 'expanded hollow button secondary login')
     end
   end
+
+  # Display field following used in the show pages
+  def display_field(label_name, value)
+    return if value.nil?
+    raw "
+    <div class='field'>
```
Comment (hapiben) [inline app/helpers/dam_uploader_helper.rb]: Can we change this to [content_tag](http://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag)?

File: app/views/dam_dashboard/index.html.erb [hunk 4]
```diff
@@ -18,6 +18,13 @@
 
 <section class="row main-section">
   <div class="medium-9 large-7 medium-centered">
-    <%= render(partial: 'dam_dashboard/resource_table') %>
+    <%= render 'dam_dashboard/resource_grid/grid', grid_title: "Need to be reviewed", dam_resources: @under_review_dam_resources if current_dam_user.admin? %>
```
Comment (hapiben) [inline app/views/dam_dashboard/index.html.erb]: Not sure if applicable. "Need to be reviewed" should be part of the translation file. Smae with the ones below.

File: app/views/dam_dashboard/resource_table/_item.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  <td>
+      <div>
+        <%= image_tag dam_resource.dam_media.media_file.thumb, class: '', size: '100x100', alt: dam_resource.dam_media.media_file.file.filename %>
+      </div>
+  </td>
+  <td>
+    <%= dam_resource.title %>
+  </td>
+  <td class="text-center row">
+    <span class="small-6 column"><%= link_to "Edit", edit_dam_resource_path(dam_resource) %></span>
```
Comment (hapiben) [inline app/views/dam_dashboard/resource_table/_item.html.erb]: "Edit", "Remove" part of translation file?

File: app/views/shared/dam/_admin_navigation.html.erb [hunk 6]
```diff
@@ -1,17 +1,22 @@
...
+      <li>
+        <%= link_to(root_path, class: 'admin-title text-center') do %>
+          Administration
+        <% end %>
+      </li>
+      <li><%= link_to "Users", dam_user_manager_index_path, class: current_class?(dam_user_manager_index_path) %></li>
+      <li>
+        <a href="#" class="">Content management</a>
+        <ul class="menu">
+          <li><%= link_to "Locations", dam_locations_path, class: current_class?(dam_locations_path) %></li>
```
Comment (hapiben) [inline app/views/shared/dam/_admin_navigation.html.erb]: Link texts in translation file?

File: app/views/shared/dam/_mobile_user_navigation.html.erb [hunk 7]
```diff
@@ -13,7 +13,6 @@
   <div class="top-bar-right">
     <ul class="vertical menu mobile-menu expanded text-center">
       <li class="menu-text"><%= link_to "Search", dam_resources_path, class: current_class?(dam_resources_path) %></li>
```
Comment (hapiben) [inline app/views/shared/dam/_mobile_user_navigation.html.erb]: "Search" in translation file.

File: app/views/dam_dashboard/index.html.erb (hunk 4 above)
Comment (motizuki) [inline app/views/dam_dashboard/index.html.erb]: Yep it does - thanks +1

**Pull request merged**